letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -212,12 +212,21 @@ def deploy_cert(self, domain, cert_path, key_path,
         logger.info("Deploying Certificate to VirtualHost %s", vhost.filep)
 
         # Assign the final directives; order is maintained in find_dir
-        self.aug.set(path["cert_path"][-1], cert_path)
-        self.aug.set(path["cert_key"][-1], key_path)
-        if chain_path is not None:
+        if self.version >= (2, 4, 8):
```

Comment:
(`letsencrypt install` needs to start using storage.py, but currently it does not).